packages/ui/src/lib/table/Table.svelte

```diff
@@ -17,10 +17,9 @@ import type { Column, Row } from './table';
 
 export let kind: string;
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let columns: Column<any>[];
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let row: Row<any>;
-export let data: { selected?: boolean; name?: string }[];
+export let columns: Column<T, any>[];
```

Comment:
I cannot remove this any because of the way we define the Columns. I did my best, but we really allow any type as `RenderType`, it default to `T`, but we can define any value, using `unknown` lead to issue, as unknown is incompatible with `T`, same for `never`.

https://github.com/podman-desktop/podman-desktop/blob/b132d30046cd484d77233a8c69e0bb02fc3ba27b/packages/ui/src/lib/table/table.ts#L90